src/init/init.ts

```diff
@@ -109,14 +115,30 @@ function getGeneratorDescription(generator: YeomanEnvironment.GeneratorMeta, gen
 }
 
 /**
- * Extract the meaningful name from the full yeoman generator name
+ * Extract the meaningful name from the full Yeoman generator name.
```

Comment:
Love the comments, can you just make sure they're all within 80 characters? Especially the `// -----` line. We're going to start automatically reformatting these and I don't want this comment to break when we do.